app/views/shared/_header.html.erb

```diff
@@ -1,4 +1,4 @@
-<header role="eui-banner">
+<header class="mmt-header" role="eui-banner">
```

Comment:
It looks like almost all of `header.mmt-header` is overwritten by `.internal header`, which explains why I couldn't find any visual difference 
![screen shot 2017-08-02 at 5 40 40 pm](https://user-images.githubusercontent.com/171388/28896307-e35d69aa-77a9-11e7-8d8c-8a1c1295bd7c.png)